beam-core/beam-core.cabal

```diff
@@ -2,7 +2,7 @@
 -- see http://haskell.org/cabal/users-guide/
 
 name:                beam-core
-version:             0.9.2.0
+version:             0.9.3.0
```

Comment:
I believe by the PVP this counts as a breaking change, even though it won't likely affect regular users. I wanted to get as much non-breaking stuff out of the backlog first before getting to breaking changes, but I think we're in a pretty good state now to start merging things for an eventual major release. Can you bump to 0.10?

Reply:
You're right!